### PR TITLE
Added new tutorial on managing galaxy on k8s

### PR DESCRIPTION
**Pull request by nuwang**

### CODE REVIEW

**Comment by erasche** [comment]
Hey @nuwang is this for GCC?

**Comment by nuwang** [comment]
@erasche Yes, for the k8s training session.

**Comment by erasche** [comment]
Ahh ok that's unfortunate. I probably can find time to review this on
Monday, we had really hoped to have everyone's PRs in by last wednesday.

Op za 29 jun. 2019 13:59 schreef Nuwan Goonasekera <notifications@github.com

**Comment by nuwang** [comment]
That'd be really helpful! thanks!

File: topics/admin/tutorials/k8s-managing-galaxy/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+> ### Agenda
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Prerequisites
+This tutorial builds on the material of the previous
+[tutorial]({{ site.baseurl }}/topics/admin/tutorials/galaxy-on-k8s/tutorial.md)
```
Comment (erasche) [inline topics/admin/tutorials/k8s-managing-galaxy/tutorial.md]: should be tutorial.html (or use the link yn

**Comment by erasche** [comment]
Should this be merged into master?

**Pull request merged**